test/e2e/specs/editor/various/popovers.spec.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'new editor state', () => {
```

Comment:
Looks like just a small copy-paste error :)

```suggestion
test.describe( 'popovers', () => {
```